config.h.in

```diff
@@ -1 +1,2 @@
 #cmakedefine NO_XSIMD
+#cmakedefine _HAS_AUTO_PTR_ETC 0
```

Comment:
What does this line do? Is for the first issue you mentioned, of the conda build failing on mac?

Reply:
It is defining the macro _HAS_AUTO_PTR_ETC as 0. Boost (functional.hpp) does not check for deprecated C++ functions if the macro is defined with a value 0.